Automattic/wp-calypso
# BackupBrowser: release granular restore for multiple files

## Proposed Changes

* Add `calypso_jetpack_backup_browser_restore_multiple_files` event to track when a user clicks on `Restore selected files` button in the backup browser.
* Add `calypso_jetpack_backup_browser_download_multiple_files` event to track when a user clicks on `Download selected files` button in the backup browser.
* Remove `jetpack/backup-granular` feature flag.

## Testing Instructions
* Spin up a Calypso or Jetpack Cloud live branch
* Select a site with a Jetpack VaultPress Backup plan
* Pick a backup and click on `View files` inside the `Actions (+)` menu
* Ensure you see the checkboxes at the left of the files/directories
* Try to select some files and then click on `Restore selected files` on top. It should show you the restore confirmation page.
* Try to select some files and then click on `Download selected files` on top. It should redirect you to the download page.

## Pre-merge Checklist
